tests/openshift/multinetwork.yml

```diff
@@ -0,0 +1,69 @@
+- hosts: masters[0]
```

Comment:
Can we have a description of what this tests in this file or in the README or something?